controllers/error.controller.js

```diff
@@ -2,6 +2,6 @@ exports.handle404Paths = (req, res, next) => {
   res.status(404).send({ msg: "path not found" });
 };
 
-exports.handle500Paths = (req, res, next) => {
+exports.handle500Paths = (err, req, res, next) => {
   res.status(500).send({ msg: "internal server error" });
```

Comment:
I would recommend including a console.log here to log the `err` before you send the response - if the error ends up in this 500 block then it's an error you weren't expecting to get and it's really helpful to see a log of what the error was if that happens